include/lsst/afw/geom/SpherePoint.h

```diff
@@ -64,6 +64,13 @@ class SpherePoint
 {
 public:
     /**
+     * Construct a SpherePoint with longitude, latitude = 0, 0
+     *
+     * @exceptsafe Provides strong exception guarantee.
+     */
+    SpherePoint();
```

Comment:
I'd still prefer not to let `SpherePoint` be default-constructible if it can be avoided; the stated default, while better than any of the alternatives, is not as obvious as it could be because the default point is `(0, 0)` in lon-lat representation but `(1, 0, 0)` in vector representation.

Do you use default constructibility anywhere? I tried looking for explicit variable declarations but didn't find any.

If you do keep the default constructor, consider declaring it `noexcept`.